Gordon360/Services/ProfileService.cs

```diff
@@ -359,6 +368,50 @@ public async Task UpdateImagePrivacyAsync(string username, string value)
 
             return profile.ToObject<ProfileViewModel>();
         }
+        public void SendUpdateEmail(string username, string email_content)
+        {
+            using (var smtp = new SmtpClient())
+            {
+                var account = _accountService.GetAccountByUsername(username);
+                //string to_email = "devrequest@gordon.edu";
+                /*
+                 * TO EMAIL TO BE CHANGED TO DEVREQUEST (commented above) change to_email variable for testing
+                 */
+                string bcc_email =  _context.ACCOUNT.FirstOrDefault(x => x.gordon_id == account.GordonID)?.email ?? ""; 
+                string to_email = "devrequest_placeholder@gordon.edu";
```

Comment:
We shouldn't commit this to train with an email that doesn't exist.

Additionally, perhaps it's a larger change, but this address might best be placed in the appsettings files as well. @EjPlatzer what do you think.

Reply:
Extra layers of security can't hurt. I'm cool for implementing this if @EjPlatzer is good with it